mrp_production_filter_lot/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     'version': '12.0.1.0.1',
     'category': 'Manufacturing',
     'website': "https://github.com/OCA/manufacture"
-               "12.0/mrp_production_filter_lot",
+               "/tree/12.0/mrp_production_filter_lot",
```

Comment:
Correct link should be only `https://github.com/OCA/manufacture`

Reply:
Thanks for having a look!
In https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#11modules, both forms are accepted:
> The website key must be [https://github.com/OCA/<repo](https://github.com/OCA/%3Crepo)>, so as to provide the most relevant link to discover more information about the addon. That link shows the repository README. Alternatively [https://github.com/OCA/<repo>/tree/<branch>/<addon](https://github.com/OCA/%3Crepo%3E/tree/%3Cbranch%3E/%3Caddon)> may be used, to provide a direct link to the addon README, which includes proper credits (authors, contributors and theirs companies), and links to the relevant information on the OCA website.

And personally, I prefer the link to the module because I think that if a user is clicking on a link in a module it is expecting information about the module itself and not information about a repository that happens to contain the module.